packages/client/hmi-client/src/components/funman/tera-constraint-group-form.vue

```diff
@@ -142,14 +142,16 @@ watch(
 
 <style>
 .constraint-group {
-	margin: 1rem;
+	width: 100%;
+	margin-top: 1rem;
 	display: flex;
 	padding: 1rem 1rem 1rem 1.5rem;
 	flex-direction: column;
 	justify-content: center;
 	align-items: flex-start;
 	border-radius: 0.375rem;
 	background: #fff;
+	border: 1px solid rgba(0, 0, 0, 0.08);
 	/* Shadow/medium */
 	box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.06), 0px 4px 6px -1px rgba(0, 0, 0, 0.08);
```

Comment:
We have CSS variables for that: https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L863-L950

Reply:
Thanks i missed this first go through of the variables.css i will make a new PR for this cleanup